scripts/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+mkdir -p /tmp/explore-logs/config
+mkdir -p /tmp/explore-logs/provisioning/datasources
+mkdir -p /tmp/explore-logs/provisioning/plugins
+
+# Download files
+curl https://raw.githubusercontent.com/grafana/explore-logs/main/config/loki-config.yaml -o /tmp/explore-logs/config/loki-config.yaml
+curl https://raw.githubusercontent.com/grafana/explore-logs/main/provisioning/datasources/default.yaml -o /tmp/explore-logs/provisioning/datasources/default.yaml
+curl https://raw.githubusercontent.com/grafana/explore-logs/main/provisioning/plugins/app.yaml -o /tmp/explore-logs/provisioning/plugins/app.yaml
+
+curl https://raw.githubusercontent.com/grafana/explore-logs/main/docker-compose.yaml -o /tmp/explore-logs/docker-compose.yaml
+
```

Comment:
Locally I changed this to `cp ../docker-compose.yaml -o /tmp/explore-logs/` but I still see
![Screenshot 2024-04-09 at 1 54 40 PM](https://github.com/grafana/explore-logs/assets/114438185/61335113-c952-44e1-b8c9-60c801fe7583)

Reply:
Should work with `cp ../docker-compose.yaml /tmp/explore-logs/`. Note the line in your output:
```
cp: -o: No such file or directory
```

Reply:
Yeah I might be having some strange permission issues according to google.

Reply:
Did you remove the `-o` from the `cp` command you ran?